### DISCUSSION TITLE
Configuring module bar per role

### DISCUSSION THREAD

**Discussion by iqraaaswad** [directus/directus#9076] [Feature Requests]
is it possible to customize module bar using "Role and Permission" section... so we can provide different module bar per user / role..

**Comment by benhaynes** [comment]
Not as of now. This was moved to a GLOBAL configuration, and we'll add it back to Roles in the future.

**Reply by aaron-manning** [reply]
Is there any timeframe for this feature to be re-implemented?

I'm looking to use Directus for a future project and am saddened to see this feature was removed from the RC.

**Reply by benhaynes** [reply]
No, not as of now. We're going to focus on stability for the next few weeks, so at least not until then, unless it gets sponsored.

**Reply by zbjdonald** [reply]
I sponsor 5$ per month for this feature.

**Reply by zbjdonald** [reply]
Any updates T。T?

**Reply by rijkvanzanten** [reply]
> Any updates T。T?

This enhancement isn't getting a lot of traction from the community, so it's not something we're currently aiming to implement.

**Reply by eeerdem** [reply]
Actually, this is what I am waiting for to finalize my project. It might be very usefull to open/close import and export button base on user role.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
